sdk/synapse/Azure.Analytics.Synapse.AccessControl/CHANGELOG.md

```diff
@@ -2,24 +2,24 @@
 
 ## 1.0.0-preview.6 (2022-06-07)
 
-### Changed
+### Features Added
```

Comment:
I updated  `changelog.md` to comply the changelog guideline to pass ci. The changes are not caused by the pr https://github.com/Azure/autorest.csharp/pull/4790.